e2e/test_driver.sh

```diff
@@ -16,10 +16,10 @@ esac
 timeout 2m flower-server $server_arg &
 sleep 3
 
-timeout 2m flower-client $client_arg --callable client:flower --server 127.0.0.1:9092 &
+timeout 2m flower-client --insecure client:flower $client_arg --server 127.0.0.1:9092 &
```

Comment:
What does the `$client_arg` do?

Reply:
`$client_arg` is either `--insecure` or `--root-certificates xxx`. The `--insecure` in the command is redundant. I will remove it.